### PR TITLE
[PBW-4980] Track video instance by player

### PR DESCRIPTION
**Pull request by angieisrani**

### CODE REVIEW

File: src/main/js/main_html5.js [hunk 1]
```diff
@@ -70,10 +70,11 @@ require("../../../html5-common/js/utils/environment.js");
      * @param {string} domId The dom id of the video player instance to create
      * @param {object} controller A reference to the video controller in the Ooyala player
      * @param {object} css The css to apply to the video element
+     * @param {string} playerId An id that represents the player instance
      * @returns {object} A reference to the wrapper for the newly created element
      */
-    this.create = function(parentContainer, domId, controller, css) {
-      if (this.maxSupportedElements > 0 && currentInstances >= this.maxSupportedElements) {
+    this.create = function(parentContainer, domId, controller, css, playerId) {
```
Comment (aldigun) [inline src/main/js/main_html5.js]: In the case of CMS-less, playerId can be null.

Comment (angieisrani) [inline src/main/js/main_html5.js]: You may be thinking of player branding id?  This should be player container in DOM

Comment (aldigun) [inline src/main/js/main_html5.js]: okay, this is not playerBrandingId, so ignore this message.

File: src/main/js/main_html5.js [hunk 2]
```diff
@@ -70,10 +70,13 @@ require("../../../html5-common/js/utils/environment.js");
...
+      // If the current player has reached max supported elements, do not create a new one
+      if (this.maxSupportedElements > 0 && playerId &&
```
Comment (angieisrani) [inline src/main/js/main_html5.js]: @aldigun  added additional check here for playerId, ready for review

**Comment by aldigun** [comment]
Looks good

**Pull request merged**